src/gedcom.ts

```diff
@@ -172,16 +218,26 @@ function createIndi(
   }
 
   // Image URL.
-  const objeTag = findTag(entry.tree, 'OBJE');
-  if (objeTag) {
+  const objeTags = findTags(entry.tree, 'OBJE');
+  if (objeTags.length > 0) {
     // Dereference OBJEct if needed.
-    const realObjeTag = objeTag.data
-      ? objects.get(pointerToId(objeTag.data))!
-      : objeTag;
-    const fileTag = realObjeTag && findTag(realObjeTag.tree, 'FILE');
-    if (fileTag) {
-      indi.imageUrl = fileTag.data;
-    }
+    const getFileTag = (tag: GedcomEntry) => {
+      const realObjeTag = tag.data ? objects.get(pointerToId(tag.data)) : tag;
+      if (!realObjeTag) return undefined;
+
+      const file = findTag(realObjeTag.tree, 'FILE');
+      const title = findTag(realObjeTag.tree, 'TITL');
+
+      if (!file) return undefined;
+      return {
+        url: file.data,
+        title: title && title.data,
+      } as JsonImage;
+    };
+
+    indi.images = objeTags
+      .map(getFileTag)
+      .filter((x): x is JsonImage => x !== undefined);
```

Comment:
This should work:
```typescript
.filter((x) => !!x);
```

Reply:
that's really ugly part. did not find a better option. eventually, this example does not work